device/delta/x86_64-delta_agc032-r0/fancontrol.service

```diff
@@ -0,0 +1,295 @@
+
+	# Check whether all mandatory settings are set
+	if [[ -z ${INTERVAL} || -z ${FCTEMPS} || -z ${MINTEMP} || -z ${MAXTEMP} || -z ${MINSTART} || -z ${MINSTOP} ]]
+	then
+		echo "Some mandatory settings missing, please check your config file!" >&2
+		exit 1
+	fi
+	if [ "$INTERVAL" -le 0 ]
+	then
+		echo "Error in configuration file:" >&2
+		echo "INTERVAL must be at least 1" >&2
+		exit 1
+	fi
+
+	# write settings to arrays for easier use and print them
+	echo
+	echo "Common settings:"
+	
+	temp_string=$FCTEMPS
+	
+	let fcvcount=0
+	for fcv in $FCTEMPS
+	do
+		fcvcount=$((fcvcount+1))
+		AFCTEMP[$fcvcount]=$( echo "$temp_string" | cut -d" " -f $fcvcount )
+		AFCTEMP[$fcvcount]=$( echo "${AFCTEMP[$fcvcount]}" | sed 's/hwmon1/\/sys\/bus\/i2c\/devices/g' )
+		AFCTEMP_PATH[$fcvcount]=$( echo "${AFCTEMP[$fcvcount]}" | sed 's/hwmon1/\/sys\/bus\/i2c\/devices/g' )
+        AFCTEMP[$fcvcount]=$( cat ${AFCTEMP[$fcvcount]} )
+		AFCTEMP[$fcvcount]=$(( AFCTEMP[$fcvcount]/1000 ))
+	done
+
+	fan_string=$FCFANS
+	fcvcount=0
+	zero=0
+	for fcv in $FCFANS
+	do
+		fcvcount=$((fcvcount+1))
+		AFCFAN[$fcvcount]=$( echo "$fan_string" | cut -d" " -f $fcvcount )
+		AFCFAN_PATH[$fcvcount]=$( echo "${AFCFAN[$fcvcount]}" | sed 's/hwmon1/\/sys\/bus\/i2c\/devices/g' )
+		AFCFAN_TARGET[$fcvcount]=$( echo "${AFCFAN_PATH[$fcvcount]}" | sed 's/hwmon1/\/sys\/bus\/i2c\/devices/g' )
+		AFCFAN_TARGET[$fcvcount]=$( echo "${AFCFAN_TARGET[$fcvcount]}"  | sed 's/$/_percentage/g')
+		AFCFAN[$fcvcount]=$( cat ${AFCFAN_PATH[$fcvcount]} )
+		if [ "${AFCFAN[$fcvcount]}" == 960 ]
+		then
+			AFCFAN[$fcvcount]=$zero
+		fi
+	done
+}
+
+# Check that all referenced sysfs files exist
+function CheckFiles
+{
+	local outdated=0 fcvcount tsen fan
+	if [ $outdated -eq 1 ]
+	then
+		echo >&2
+		echo "At least one referenced file is missing. Either some required kernel" >&2
+		echo "modules haven't been loaded, or your configuration file is outdated." >&2
+		echo "In the latter case, you should run pwmconfig again." >&2
+	fi
+	return $outdated
+}
+
+#LoadConfig $1
+if [ -f "$1" ]
+then
+    LoadConfig $1
+else
+    LoadConfig /etc/fancontrol
+fi
+
+# Detect path to sensors
+if [ ! -d $DIR ]
+then
+	echo $0: 'No sensors found! (did you load the necessary modules?)' >&2
+	exit 1
+fi
+cd $DIR
+
+# Check for configuration change
+if [ "$DIR" != "/" ] && [ -z "$DEVPATH" -o -z "$DEVNAME" ]
+then
+	echo "Configuration is too old, please run pwmconfig again" >&2
+	exit 1
+fi
+if [ "$DIR" = "/" -a -n "$DEVPATH" ]
+then
+	echo "Unneeded DEVPATH with absolute device paths" >&2
+	exit 1
+fi
+CheckFiles || exit 1
+
+if [ -f "$PIDFILE" ]
+then
+	echo "File $PIDFILE exists, is fancontrol already running?" >&2
+	exit 1
+fi
+echo $$ > "$PIDFILE"
+
+# main function
+function UpdateThermalSensors
+{
+	fcvcount=0
+	for fcv in $FCTEMPS
+	do
+		fcvcount=$((fcvcount+1))
+        AFCTEMP[$fcvcount]=$( cat ${AFCTEMP_PATH[$fcvcount]} )
+		AFCTEMP[$fcvcount]=$(( AFCTEMP[$fcvcount]/1000 ))
```

Comment:
Inconsistent indentation. Makes reading the code difficult. Can you please replace all tabs with four spaces throughout the files and make indentation consistent?

Reply:
@jleveque 
Sorry for the inconvenience, 
i have replaced tab with 4 spaces and commit it, thank you